lib/service_check.pm

```diff
@@ -250,18 +257,18 @@ sub check_services {
         my $srv_pkg_name  = $service->{$s}->{srv_pkg_name};
         my $srv_proc_name = $service->{$s}->{srv_proc_name};
         my $support_ver   = $service->{$s}->{support_ver};
+        my $service_type  = 'Systemd';
         next unless _is_applicable($srv_pkg_name);
         record_info($srv_pkg_name, "service check after migration");
         eval {
             if (is_sle($support_ver, $hdd_base_version)) {
                 # service check after migration. if we've set up service check
                 # function, we don't need following actions to check the service.
                 if (exists $service->{$s}->{service_check_func}) {
-                    $service->{$s}->{service_check_func}->();
+                    $service->{$s}->{service_check_func}->('after', 'Systemd');
```

Comment:
Why not use $service_type?

Reply:
for avoid using check_version everywhere.